Tests/Tests/DatastoreConflictResolvers.m

```diff
@@ -188,8 +188,8 @@ -(id) init
 -(CDTDocumentRevision *)resolve:(NSString *)docId conflicts:(NSArray *)conflicts
 {
     CDTMutableDocumentRevision * mutableRev = [CDTMutableDocumentRevision revision];
-    mutableRev.body = @{};
-    mutableRev.attachments = @{};
+    mutableRev.body = [NSMutableDictionary dictionary];
```

Comment:
These lines can now be removed because body and attachments will already have been constructed